src/Microsoft.AspNetCore.Mvc.Testing/MvcWebApplicationBuilder.cs

```diff
@@ -56,7 +57,7 @@ public MvcWebApplicationBuilder<TStartup> ConfigureAfterStartup(Action<IServiceC
         public MvcWebApplicationBuilder<TStartup> UseApplicationAssemblies()
         {
             var depsFileName = $"{typeof(TStartup).Assembly.GetName().Name}.deps.json";
-            var depsFile = new FileInfo(Path.Combine(Directory.GetCurrentDirectory(), depsFileName));
+            var depsFile = new FileInfo(Path.Combine(AppContext.BaseDirectory, depsFileName));
             if (!depsFile.Exists)
             {
                 throw new InvalidOperationException($"Can't find'{depsFile}'. This file is required for functional tests " +
```

Comment:
This error message needs to mention shadow copying. It needs to explain that if your tests aren't running from where your build output is, you need to fix that - or you need to make sure your .deps file gets copied too